roles/foreman_repositories/README.md

```diff
@@ -0,0 +1,40 @@
+theforeman.operations.foreman_repositories
+==========================================
+
+Sets up the Foreman and plugin repositories.
+
+Role Variables
+--------------
+
+Optional:
+
+- `foreman_repositories_version`: Version of Foreman to setup repositories for (default: 2.3)
+- `foreman_repositories_katello_version`: Version of Katello to setup repositories for (default: )
```

Comment:
This should somehow tell the user that if they don't set a version, no Katello repos are configured.

Reply:
Check the wording now